libs/ui/search/src/lib/results-table/results-table.component.ts

```diff
@@ -48,11 +41,14 @@ export class ResultsTableComponent {
   @Input() selectedRecordsIdentifiers: string[] = []
   @Input() sortOrder: SortByField = null
   @Input() recordHasDraft: (record: CatalogRecord) => boolean = () => false
+  @Input() recordIsDraft: (record: CatalogRecord) => boolean = () => false
```

Comment:
What is the difference between `recordHasDraft` and `recordIsDraft`?

Reply:
`recordHasDraft` means a record with a draft (ie. changed locally but not published), `recordIsDraft` means a draft without record (ex: new record not yet published, duplicate record not yet published).

But you're right, this is not clear enough, I'll change the name.

Reply:
I see! Maybe `draftWithoutRecord`?

Reply:
I went for `hasDraft` (can only be a record then), `isDraft` and `isDraftWithoutRecord`.